draft-ietf-core-dns-over-coap.md

```diff
@@ -388,7 +388,7 @@ The same applies for DNS responses that for any reason do not carry any records
 
 The following example illustrates the response to the query "example.org. IN
 AAAA record", with recursion turned on. Successful responses carry one answer
-record including address 2001:db8:1::1:2:3:4 and TTL 79689.
+record including address 2001:db8:1:0:1:2:3:4 and TTL 79689.
```

Comment:
Probably an opportunity for me to learn, but why is double colon `::` not possible here?

Reply:
a double colon represents consecutive blocks of zeros, but in this case there block size is 1.